test/mix/tasks/exfmt_test.exs

```diff
@@ -4,33 +4,43 @@ defmodule Mix.Tasks.ExfmtTest do
 
   describe "mix exfmt" do
     test "with no args" do
-      io = capture_io(fn->
+      io = capture_io(:stderr, fn->
```

Comment:
I left a few cases inside this test to serve as an "integration" test of sorts, to verify the mix command itself functions